contributors/devel/sig-scheduling/scheduler_benchmarking.md

```diff
@@ -0,0 +1,77 @@
+# Scheduler Benchmarking
+
+Kubernetes scheduler has integration and e2e benchmarks. It is recommended to
+run integration benchmarks at the time of submitting PRs that could potentially
+impact performance of the scheduler and give before and after results with your
+PR submission.
+
+## Running integration benchmarks
+
+To run integration benchmarks use the following command from inside a Kubernetes
+directory. 
+
+```sh
+make test-integration WHAT=./test/integration/scheduler_perf KUBE_TEST_VMODULE="''" KUBE_TEST_ARGS="-run=xxx -bench=BenchmarkScheduling"
```

Comment:
how about `-bench=.` ?

Reply:
Added it and more information about Go benchmarks.

Reply:
Did you mean to remove this now that you added the other example?

Reply:
No, not really. I actually wanted to keep this so that people find an example of how particular benchmarks can be specified in CLI.